tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_Translation.cpp

```diff
@@ -209,13 +246,41 @@ void test_translation() {
     // jacobian to be computed in the wrong region. However, when it calls the
     // jacobian, it still uses the original point_xi so there's a mixup between
     // which region it's in.
-    if (inspiral_radius <= inner_radius * .99 or
-        inspiral_radius >= inner_radius * 1.01) {
+    if (piecewise_radius <= inner_radius * .99 or
+        piecewise_radius >= inner_radius * 1.01) {
       test_jacobian(piecewise_translation_map, point_xi, t, f_of_t_list);
       test_inv_jacobian(piecewise_translation_map, point_xi, t, f_of_t_list);
+      test_jacobian(piecewise_translation_map_deserialized, point_xi, t,
+                    f_of_t_list);
+      test_inv_jacobian(piecewise_translation_map_deserialized, point_xi, t,
+                        f_of_t_list);
+    }
+    // Checking if any radius in the datavector is within the range above.
+    bool jac_flag = true;
+    for (size_t i = 0; i < get_size(get_element(point_xi_dv, 0)); i++) {
```

Comment:
Use `gsl::at` when the type is known to be an `array`, instead of `get_element`.
Use `.size()` instead of `get_size` when the type is known to be a container.